### PR TITLE
elastic stack: bump to 7.9.2

### PR DESCRIPTION
**Pull request by hydrapolic**
Copy of 7.9.1, tested on amd64.

### CODE REVIEW

File: app-misc/elasticsearch/elasticsearch-7.9.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+	fowners root:${PN} /etc/${PN}
+	fperms 2750 /etc/${PN}
+
+	insinto /usr/share/${PN}
+	doins -r .
+
+	exeinto /usr/share/${PN}/bin
+	doexe "${FILESDIR}/elasticsearch-systemd-pre-exec"
+
+	chmod +x "${ED}"/usr/share/${PN}/bin/* || die
```
Comment (juippis) [inline app-misc/elasticsearch/elasticsearch-7.9.2.ebuild:47]: Switch to `fperms`.

File: app-misc/elasticsearch/elasticsearch-7.9.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+	if use x-pack; then
+		chmod +x "${ED}"/usr/share/${PN}/modules/x-pack-ml/platform/linux-x86_64/bin/* || die
```
Comment (juippis) [inline app-misc/elasticsearch/elasticsearch-7.9.2.ebuild:50]: Here too, `fperms`. It doesn't need to `|| die` in an ebuild.

File: app-misc/elasticsearch/elasticsearch-7.9.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	dodir /usr/share/${PN}/plugins
+
+	insinto /etc/sysctl.d
+	newins "${FILESDIR}/${PN}.sysctl.d" ${PN}.conf
+
+	newconfd "${FILESDIR}/${PN}.conf.3" ${PN}
+	newinitd "${FILESDIR}/${PN}.init.7" ${PN}
+
+	systemd_install_serviced "${FILESDIR}/${PN}.service.conf"
+	systemd_newtmpfilesd "${FILESDIR}/${PN}.tmpfiles.d" ${PN}.conf
```
Comment (juippis) [inline app-misc/elasticsearch/elasticsearch-7.9.2.ebuild:65]: This is deprecated and now done via *tmpfiles.eclass*. Do note the new `pkg_postinst` function about it too.

**Comment by hydrapolic** [comment]
Thanks @juippis, pushed an update.

**Review by juippis — APPROVED** [review]

**Comment by hydrapolic** [comment]
Thanks @juippis

**Pull request closed without merging**